### PR TITLE
Feature/#043 test registro

### PR DESCRIPTION
**Pull request by juradosalado**

### CODE REVIEW

File: innoweb/test.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from participant.models import Participant
+from event.models import Event
+# from selenium.webdriver.chrome.webdriver import WebDriver
```
Comment (migromarj) [inline innoweb/test.py:5]: You should remove the commented import

**Review by migromarj — COMMENTED** [review]
You should remove the chromedriver executable as that should not be in the project folder, I have also made a suggestion to remove an import that is commented out.

**Review by TomasCB24 — APPROVED** [review]
You have commented import if I'm not mistaken. Take that away and everything would be perfect. Good job.

**Review by migromarj — APPROVED** [review]
Thanks for considering the suggestions! All tests run as expected, great job Juan 💪🏻 🙌🏻

**Pull request merged**